platform-sdk/swirlds-logging/src/test/java/com/swirlds/logging/util/HandlerLoggingLevelConfigTestOrchestrator.java

```diff
@@ -75,7 +75,6 @@ private HandlerLoggingLevelConfigTestOrchestrator(TestScenario... testScenarios)
      * Performs the verification of scenario given by its index on the list
      */
     private void testScenario(int scenario, HandlerLoggingLevelConfig config) {
-        System.out.printf("Testing scenario %d%n", scenario);
```

Comment:
I do not get it. It can't be a problem to do 1 line of output per test (why I would like to use the emergency logger instead of sys out)

Reply:
It is not, and it also doesn't add any value. And when a scenario fails, it does say which one.
I didn't use the emergency logger, so we don't have circular dependencies. The tool is a helper for testing the logging framework, and I didn't want it to use the logging framework for its internal activities.